docs/src/man/categorical.md

```diff
@@ -77,23 +77,33 @@ julia> sort(cv)
 
 ```
 
-By default, a `CategoricalArray` is able to represent 2<sup>32</sup>differents levels. You can use less memory by calling the `compress` function:
+By default, a `CategoricalArray` is able to represent 2<sup>32</sup> different levels. You can use less memory by calling the `compress` function, or by calling the `categorical` function with the `compress` argument set to `true`:
```

Comment:
Can you please break this line as it seems overlong?
Also I would add that `compess` is a positional argument for clarity.